### ISSUE TITLE
disable on em-text has no effect

### ISSUE THREAD

**Issue by joshpfosi** [piceaTech/ember-rapid-forms#27]
Setting the `disabled=true` flag on the `em-text` component seems to be dysfunctional e.g.:

```html
{{#em-form model=answer submit_button=false}}
  {{em-text rows="2" property="text" disabled=true}}
{{/em-form}}
```

It appropriately sets the disabled flag in the component but the input field does not carry the `disabled=true` binding.

**Comment by pedrokiefer** [comment]
`Ember.TextArea` by default doesn't bind the disabled attribute, not sure why. I'll add it here, but it might be good to also do a PR for ember, it's a one line change.

**Comment by joshpfosi** [comment]
Just made a PR #28 which fixes this with a test.

**Comment by pedrokiefer** [comment]
Fixed by #28
